check_sites.py

```diff
@@ -100,16 +111,19 @@ def main():
                     "\nerror was: " + inst)
                 return
         old_checker = FpsCheck(old_sites, etlds, icanns)
-        check_sets, _ = find_diff_sets(old_checker.load_sets(), fps_checker.load_sets())
+        check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), fps_checker.load_sets())
         # TODO: add variable and check for subtracted_sets in case of user 
         # removing old set from the list
     else:
         check_sets = fps_checker.load_sets()
```

Comment:
We also need to assign `subtracted_sets` here, right? This is the case when we're checking the entire list's invariants (not just the diff), which means we ought to consider what was removed too.

Reply:
I don't think so? In this case, there is no sense of an old list so there is nothing to check against to determine if an element was removed